src/components/Hero/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { View, ImageBackground, StyleSheet } from 'react-native';
+import { Config } from 'react-native-config';
+import LinearGradient from 'react-native-linear-gradient';
+import Labels from './Labels';
+import SpecialBanner from './SpecialBanner';
+import Controls from './Controls';
+import { Header } from '@/components';
+import { spacing } from '@/theme';
+import { getTrending, getAllGenre } from '@/controllers/MovieController';
+
+export const Hero = () => {
+  const [posterURL, setPosterURL] = useState('');
+  const [genreIds, setGenreIds] = useState([]);
+  const [genres, setGenres] = useState([]);
+
+  const getData = useCallback(async () => {
+    try {
+      const trendingMovies = await getTrending();
```

Comment:
In this onboarding, this is not so important because we only have 2 calls to an API in this method. But imagine you have 8 calls and those do not depend on each other. Do you know a way to make the program "faster"?